docs/spec/auth/scope.md

```diff
@@ -91,8 +91,7 @@ separator               := /[_.]|__|[-]*/
 ```
 Full reference grammar is defined
 (here)[https://godoc.org/github.com/docker/distribution/reference]. Currently
-the scope name grammar is a subset of the reference grammar without support
-for hostnames.
+the scope name grammar is a subset of the reference grammar with optional support for hostnames.
```

Comment:
The reference grammar allows optional hostnames, so I think the last part is redundant.